### PR TITLE
Fix all existing CI issues

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

File: build-jellyfin.ps1 [hunk 1]
```diff
@@ -82,14 +86,15 @@ function Install-NSSM {
...
     Write-Verbose "Checking Architecture"
     if($Architecture -notin @('x86','x64')){
         Write-Warning "No builds available for your selected architecture of $Architecture"
         Write-Warning "NSSM will not be installed"
     }else{
          Write-Verbose "Downloading NSSM"
          # [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-         # Temporary workaround, file is hosted in an azure blob with a custom domain in front for brevity
+         # File is hosted in an azure blob with a custom domain in front for brevity
          Invoke-WebRequest -Uri http://files.evilt.win/nssm/nssm-2.24-101-g897c7ad.zip -UseBasicParsing -OutFile "$tempdir/nssm.zip" | Write-Verbose
```
Comment (JustAMan) [inline build-jellyfin.ps1:98]: @anthonylavado just thinking out loud - maybe we should include that zip file in this repo?

Comment (anthonylavado) [reply build-jellyfin.ps1:98]: Yeah, we should probably move it in now.

Comment (JustAMan) [reply build-jellyfin.ps1:98]: Not sure why, given how bad nssm site stability is and that this mirror could also come and go (AFAIR it was introduced by @LogicalPhallacy who's not very active recently).

We can also bump nssm version while we're at it.

Comment (dkanada) [reply build-jellyfin.ps1:98]: We can also just throw it on our release website and achieve the same result.

Comment (JustAMan) [reply build-jellyfin.ps1:98]: Yeah, that also works

Comment (dkanada) [reply build-jellyfin.ps1:98]: [done](https://repo.jellyfin.org/releases/other)

Comment (dkanada) [reply build-jellyfin.ps1:98]: we can update the link in our next pr

**Pull request merged**